internal/graph/dotgraph.go

```diff
@@ -472,3 +472,13 @@ func min64(a, b int64) int64 {
 	}
 	return b
 }
+
+// escapeForDot escapes double quotes and backslashes, and replaces Graphviz's
+// "center" character (\n) with a left-justified character.
+// See https://graphviz.org/doc/info/attrs.html#k:escString for more info.
+func escapeForDot(s []string) []string {
```

Comment:
Could this function construct and return a new slice? It's current signature suggests that it does that (because it returns a slice) but in fact it changes the input slice in place. The combination of in-place modification and returning a value is imo not good. Also, I can't say just from this local context if it's OK to modify caller's labels slice content, so I'd prefer not modifying its contents.

Reply:
Yup, makes sense. Done.